src/mixins/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/mixins",
-  "version": "1.0.8",
+  "version": "1.0.9",
```

Comment:
You don't need to update it manually. https://github.com/bcgov/bcrs-shared-components#14-update-your-main-branch-and-have-lerna-update-the-versions-of-the-affected-components this step will update it

Reply:
Done!